### DISCUSSION TITLE
ES6 fn syntax for actions like getters (question)

### DISCUSSION THREAD

**Discussion by stephenjason89** [vuejs/pinia#1477] [Help and Questions] [answered]
### Reproduction

Install @pinia/nuxt 0.2.0 together with pinia 2.0.15

### Steps to reproduce the bug

This works
<img width="508" alt="image" src="https://user-images.githubusercontent.com/3810177/180700886-f55b8029-bc56-4560-8b73-04a20fac5903.png">

but this doesn't work
<img width="490" alt="image" src="https://user-images.githubusercontent.com/3810177/180700943-e9a3e7bb-a4b5-4cfd-bb05-2876620ada09.png">

Would it be possible to support this syntax for actions?

### Expected behavior

should still work

### Actual behavior

not working

**Comment by LinusBorg** [comment, marked as answer]
That's technically impossible because you can't change what `this`refers to in an arrow function.

**Reply by stephenjason89** [reply]
Thank you
